handbook/openid-connect/openid-connect.md

```diff
@@ -0,0 +1,295 @@
+# Single Sign-on with OpenID Connect
+
+You may configure SystemLink to use [OpenID Connect](https://openid.net) for user authorization. This enables SystemLink to leverage corporate single sign-on (SSO) and the additional security benefits it provides such as streamlined login and limiting the proliferation of user credentials. This also enables SystemLink to utilize a common identity for users across multiple applications. OpenID Connect can be used in conjunction with or as a replacement for LDAP, Active Directory, and local Windows accounts for authentication.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2020R4 or greater. Please see [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+
+- A DNS name for the SystemLink server.
```

Comment:
Fixed! I left the periods in my workflow instructions (order lists) because I believe they are full sentences. Let me know if their lack of subject make me incorrect. 
e.g 
> 1. Log into the server running SystemLink and go to `C:\Program Files\National Instruments\Shared\Web Server\conf\openidc`.
>2. Add the configuration files to SystemLink to connect to your OpenID Connect provider.